src/django/api/views/sectors.py

```diff
@@ -1,76 +1,167 @@
+            {
+                "group_name": "Agriculture, Food & Beverage",
+                "sectors": [
+                    "Agriculture",
+                    "Animal Production",
+                    "Aquaculture",
+                    "Beverages",
+                    "Biotechnology",
+                ]
+            },
+            {
+                "group_name": "Apparel, Footwear, Textiles",
+                "sectors": [
+                    "Apparel",
+                    "Apparel Accessories",
+                    "Footwear",
+                    "Home Accessories",
+                    "Home Textiles",
+                ]
+            }
+        ]
+    """,
+)
+@api_view(['GET'])
+@throttle_classes([])
+def sectors(request):
 
+    submitted_sectors = set()
     embed = request.query_params.get('embed', None)
+    grouped = request.query_params.get('grouped', None)
+
     if embed is not None:
         contributor = request.query_params.get('contributor', None)
         if contributor is None:
             return Response(submitted_sectors)
 
         item_sectors = set(
-            FacilityListItem
-            .objects
-            .filter(
+            FacilityListItem.objects.filter(
                 status__in=[
                     FacilityListItem.MATCHED,
-                    FacilityListItem.CONFIRMED_MATCH],
+                    FacilityListItem.CONFIRMED_MATCH,
+                ],
                 source__contributor_id=contributor,
                 source__is_active=True,
-                source__is_public=True
-            ).annotate(
-                values=Func('sector', function='unnest')
-            ).values_list('values', flat=True)
+                source__is_public=True,
+            )
+            .annotate(values=Func('sector', function='unnest'))
+            .values_list('values', flat=True)
             .distinct()
         )
 
         claim_sectors = set(
-            FacilityClaim
-            .objects
-            .filter(contributor_id=contributor,
-                    status=FacilityClaimStatuses.APPROVED)
+            FacilityClaim.objects.filter(
+                contributor_id=contributor,
+                status=FacilityClaimStatuses.APPROVED,
+            )
             .annotate(values=Func('sector', function='unnest'))
             .values_list('values', flat=True)
             .distinct()
         )
 
         submitted_sectors = item_sectors.union(claim_sectors)
 
+        return Response(sorted(list(submitted_sectors)))
+
     else:
         submitted_sectors = set(
-            FacilityIndex
-            .objects
-            .annotate(all_sectors=Func(F('sector'), function='unnest'))
+            FacilityIndex.objects.annotate(
+                all_sectors=Func(F('sector'), function='unnest')
+            )
             .values_list('all_sectors', flat=True)
             .distinct()
         )
 
-    return Response(sorted(list(submitted_sectors)))
+        if grouped is not None:
```

Comment:
Shouldn't it be grouped for the embedded map as well? I might have missed it if you discussed this already.

Reply:
I raised this question at one of the standups. We decided not to include grouping for the sector filter on the embedded map because the number of sectors can vary for different contributors. For some, there may be only one sector, making the grouping redundant.

Reply:
Ok, thank you